tests/test_copy.py

```diff
@@ -673,6 +674,9 @@ def test_connection_writer(conn, format, buffer):
 
 
 @pytest.mark.slow
+@pytest.mark.skipif(
+    sys.implementation.name == "pypy", reason="depends on refcount semantics"
+)
```

Comment:
I've skipped all of the leak tests which compare reference counts.

Reply:
I think you can add a `pytest.skip()` in the `gc_count()` function so that you get all the current and future refcount tests skipped on pypy

Reply:
...Although this would end up skipping the test only after it has run, which is a waste of time.

Can you create a `gc` fixture, to return an object with methods `count()` and `collect()`, but we can just skip the test on fixture creation on pypy.

Reply:
There are some tests that will pass if we add `gc_collect()` after `del`, and a few other cases too that already use `gc_collect()`. If we move the function into the object returned by the fixture object we make skip tests that would happily work. Do you mind this? (Or we can also keep `gc_collect()` around for these few cases?)

Reply:
If tests pass reliably using `gc_collect()` maybe let's keep that function around. Thank you for the care :slightly_smiling_face:

Reply:
I'm exposing `gc_collect` as an additional fixture. This will allow changing the fixture from `gc_collect` to `gc` and the call from `gc_collect()` to `gc.collect()` if any cases do come up flaky.